fvm/handler/computation.go

```diff
@@ -101,7 +101,6 @@ func (c *computationMeteringHandler) Limit() uint64 {
 }
 
 func (c *computationMeteringHandler) AddUsed(used uint64) error {
-	used = c.computation.Used() + used
 	return c.computation.AddUsed(used)
 }
```

Comment:
nice catch but not a big issue, since this is mostly for reporting to users, cadence tracks and enforces it own computation used and calls this one once at the end of transaction execution